docs/knowledge_base/tips/big_data_models/index.md

```diff
@@ -0,0 +1,554 @@
+
+summary_statistic_type_cfg = Config.configure_data_node(id="summary_statistic_type", default_data="median")
+
+high_value_summary_df_cfg = Config.configure_data_node(id="high_value_summary_df")
+
+# -------------------- Tasks --------------------
+
+preprocess_and_score_task_cfg = Config.configure_task(
+    id="preprocess_and_score",
+    function=preprocess_and_score,
+    skippable=True,
+    input=[path_to_data_cfg],
+    output=[scored_df_cfg],
+)
+
+featurization_and_segmentation_task_cfg = Config.configure_task(
+    id="featurization_and_segmentation",
+    function=featurization_and_segmentation,
+    skippable=True,
+    input=[scored_df_cfg, payment_threshold_cfg, score_threshold_cfg],
+    output=[segmented_customer_df_cfg],
+)
+
+segment_analysis_task_cfg = Config.configure_task(
+    id="segment_analysis",
+    function=segment_analysis,
+    skippable=True,
+    input=[segmented_customer_df_cfg, metric_cfg],
+    output=[segment_result_cfg],
+)
+
+high_value_cust_summary_statistics_task_cfg = Config.configure_task(
+    id="high_value_cust_summary_statistics",
+    function=high_value_cust_summary_statistics,
+    skippable=True,
+    input=[segment_result_cfg, segmented_customer_df_cfg, summary_statistic_type_cfg],
+    output=[high_value_summary_df_cfg],
+)
+
+scenario_cfg = Config.configure_scenario(
+    id="scenario_1",
+    task_configs=[
+        preprocess_and_score_task_cfg,
+        featurization_and_segmentation_task_cfg,
+        segment_analysis_task_cfg,
+        high_value_cust_summary_statistics_task_cfg,
+    ],
+)
+```
+
+You can read more about configuring Scenarios, Tasks, and Data Nodes in the 
+[documentation here](../../../manuals/core/config/index.md).
+
+
+## 6. Scenario Creation and Execution
+
+Executing a Taipy scenario involves:
+
+- Loading the config;
+- Running the Taipy Core service; and
+- Creating and submitting the scenario for execution.
+
+Here's the basic code template:
+
+```python
+import taipy as tp
+from config import scenario_cfg  # Import the Scenario configuration
+
+if __name__ == "__main__":
+    tp.Core().run()  # Start the Core service
+    scenario_1 = tp.create_scenario(scenario_cfg)  
+
+    # Create a Scenario instance
+    scenario_1.submit()  # Submit the Scenario for execution
+
+    # Total runtime: 74.49s
+```
+
+
+### Skip unnecessary task executions
+
+One of Taipy's most practical features is its ability to skip a task execution if its output is 
+already computed. Let's explore this with some scenarios:
+
+#### Changing Payment Threshold
+
+```python
+# Changing Payment Threshold to 1600
+scenario_1.payment_threshold.write(1600)
+scenario_1.submit()
+
+# Total runtime: 31.499s
+```
+
+**What Happens:** Taipy is intelligent enough to skip Task 1 because the payment threshold only 
+affects Task 2. In this case, we see more than a 50% reduction in execution time by running 
+your pipeline with Taipy.
+
+#### Changing Metric for Segment Analysis
```

Comment:
Fifth-level title? No that was a test to see if you are reading the comment. Sould be a third-level title.

Reply:
Third I agree